src/nvim/ex_cmds.h

```diff
@@ -31,30 +32,19 @@ typedef struct {
 
 // Defs for inc_sub functionality
 
-// List of matches in a line
-#define _noop(x)
-KLIST_INIT(colnr_T, colnr_T, _noop)
-typedef klist_t(colnr_T) klist_colnr_T;
-#undef _noop
+// List of matches on a single line
+typedef kvec_t(colnr_T) kvec_colnr_T;
```

Comment:
@ZyX-I but we don't pass a pointer to the `kvec_t(colnr_T)`, it belongs to the enclosing MatchedLine struct, and we already use kvec_t without a typedef in this kind of case.